packages/reports/tests/unit/components/report-builder-test.js

```diff
@@ -0,0 +1,42 @@
+import Ember from 'ember';
+import { test, moduleForComponent } from 'ember-qunit';
+
+moduleForComponent('report-builder', 'Unit | Component | Report Builder', {
+  unit: true,
+  needs: [
+    'component:report-builder',
```

Comment:
This should be pulled in by the `moduleForComponent`. So we don't need to `need` this.